app.go

```diff
@@ -5,13 +5,15 @@ import (
 
 	"github.com/dogmatiq/dogma"
 	"github.com/dogmatiq/example/account"
+	"github.com/dogmatiq/example/debitpolicy"
```

Comment:
I wonder if this deserves its own package?  I would say it is part of `account`.

Reply:
I wasn't sure about this either. When you say its part of account, is it still its own aggregate but just under the account package?

Reply:
Yep, exactly.